features/FEATURE_BLE/targets/TARGET_NORDIC/TARGET_NORDIC_SOFTDEVICE/TARGET_NRF51/source/nRF5xGap.cpp

```diff
@@ -280,7 +280,7 @@ ble_error_t nRF5xGap::startAdvertising_(const GapAdvertisingParams &params)
 
     if (_privacy_enabled) {
         if (_peripheral_privacy_configuration.resolution_strategy != PeripheralPrivacyConfiguration_t::DO_NOT_RESOLVE) {
-            ArrayView<resolving_list_entry_t> entries = get_sm().get_resolving_list();
+            mbed::Span<resolving_list_entry_t> entries = get_sm().get_resolving_list();
```

Comment:
If there is a using declaration of `mbed::Span`; there is no need to fully qualify the class.

Reply:
Thanks, will change it